app/src/main/java/com/stevesoltys/seedvault/settings/SettingsViewModel.kt

```diff
@@ -224,12 +224,13 @@ internal class SettingsViewModel(
 
     internal fun backupNow() {
         viewModelScope.launch(Dispatchers.IO) {
+            val isAppBackupEnabled = backupManager.isBackupEnabled
             if (settingsManager.isStorageBackupEnabled()) {
                 val i = Intent(app, StorageBackupService::class.java)
-                // this starts an app backup afterwards
-                i.putExtra(EXTRA_START_APP_BACKUP, true)
+                // this starts an app backup afterwards (if enabled)
+                i.putExtra(EXTRA_START_APP_BACKUP, isAppBackupEnabled)
                 startForegroundService(app, i)
-            } else {
+            } else if (isAppBackupEnabled) {
```

Comment:
Seems unrelated to this PR sorta? But seems valid nonetheless. Is this a bugfix maybe?

Reply:
Yeah, I think I had noticed during testing and fixed along the way instead of PRing separately.